### PR TITLE
No ticket/minor hotfixes

### PR DESCRIPTION
**Pull request by k1r1k1**
* logout-button
* lottie animation start
* AuthSignOrReg component => container
* User info settings header => component
* Routing problems

### CODE REVIEW

File: src/sagas/authorizationSaga.js [hunk 1]
```diff
@@ -6,15 +6,19 @@ import {
...
 
 function* login(action) {
   try {
-    const user = yield call(signInWithGoogleAsync, action.payload);
-    yield put(actions.authActions.loginSuccess(user));
-    yield NavigatorService.navigate(APP_TAB);
+    /* console.log('signIn With GoogleAsync...'); */
+    yield call(signInWithGoogleAsync, action.payload);
+    /* console.log('Firebase sign in complete'); */
+    /* yield put(actions.authActions.loginSuccess());
```
Comment (Fedkasin) [inline src/sagas/authorizationSaga.js]: Please remove all comments with console logs from these file

File: src/screens/AuthLoadingScreen.js [hunk 2]
```diff
@@ -5,15 +5,15 @@ import {
...
+import { isLoggedIn } from '../helpers/authHelpers';
 
 class AuthLoadingScreen extends PureComponent {
   // Fetch the token from storage then navigate to our appropriate place
   componentDidMount() {
     const { navigation } = this.props;
     isLoggedIn(user => {
-      /* console.log(user); */
       navigation.navigate(user ? APP_TAB : AUTH_STACK);
+      // console.log('user updated', user); <==
```
Comment (Fedkasin) [inline src/screens/AuthLoadingScreen.js]: The same

**Review by k1r1k1 — COMMENTED** [review]
console logs cleaning

**Review by Fedkasin — APPROVED** [review]

**Comment by Fedkasin** [comment]
LGTM

**Pull request merged**